packages/edit-site/src/components/sidebar/global-styles-v2/navigation/view.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Internal dependencies
+ */
+import {
+	VIEW_COLORS,
+	VIEW_COLORS_ELEMENT,
+	VIEW_COLORS_PALETTE,
+	VIEW_ROOT,
+	VIEW_TYPOGRAPHY,
+	VIEW_TYPOGRAPHY_ELEMENT,
+} from './constants';
+import { useGlobalStylesNavigationContext } from './context';
+import GlobalStylesViewColors from '../views/colors';
+import GlobalStylesViewColorsElement from '../views/colors-element';
+import GlobalStylesViewColorsPalette from '../views/colors-palette';
+import GlobalStylesViewRoot from '../views/root';
+import GlobalStylesViewTypography from '../views/typography';
+import GlobalStylesViewTypographyElement from '../views/typography-element';
+
+const allViews = {
+	[ VIEW_ROOT ]: GlobalStylesViewRoot,
+	[ VIEW_COLORS ]: GlobalStylesViewColors,
+	[ VIEW_COLORS_ELEMENT ]: GlobalStylesViewColorsElement,
+	[ VIEW_COLORS_PALETTE ]: GlobalStylesViewColorsPalette,
+	[ VIEW_TYPOGRAPHY ]: GlobalStylesViewTypography,
```

Comment:
I may want to change the typography settings of block A I just installed. So in practice all these views could also be nested e.g: I may go to root-> view typography to change typography settings for all the site, or I may go to root -> blocks -> block a (dynamic depending on the blocks of site) -> view typography.

It would be nice to see how this system expands to this dynamic case. E.g: we may have a `const all blocks = []` that simulates the blocks on the website. And then we could simulate this "dynamic" view case.
It is not a blocker we can git it a try in a follow-up PR.